scripts/serverless.js

```diff
@@ -495,6 +497,17 @@ const processSpanPromise = (async () => {
 
     const configurationFilename = configuration && configurationPath.slice(serviceDir.length + 1);
 
+    if (standaloneCommands.includes(command)) {
+      require('../lib/cli/ensure-supported-command')(configuration);
+      await require(`../commands/${commands.join('/')}`)({
+        configuration,
+        serviceDir,
+        configurationFilename,
+        options,
+      });
+      return;
```

Comment:
We need to postpone it with exactly same steps as here: https://github.com/serverless/serverless/blob/54bb13b6a6d3fd41548615fc23f4ae4d6d663dcc/scripts/serverless.js#L517-L538

Probably cleanest way would be to do:

```javascript
if (isInteractiveSetup || standaloneCommands.includes(command)) {
  if (isInteractiveSetup) {
    // interactive command handling
  } else {
    // standalone command handling
  }
  // post process steps
  return;
}
```